optimum/intel/ipex/inference.py

```diff
@@ -29,15 +81,45 @@ def __init__(self, optimized, default):
 
     def __call__(self, *args, **kwargs):
         try:
-            return self._optimized(*args, **kwargs)
+            kwargs.pop("position_ids", None)

```

Comment:
It's because inputs of some models like GPT2 have `position_id` which is not contained in the jit traced model. GPT2 can use this optimization by removing `position_ids`. I can remove `kwargs.pop("position_ids", None)` and models like GPT2 will not use the optimization.